docs/src/components/Header.astro

```diff
@@ -1,56 +1,117 @@
+        height="24"
+        viewBox="0 0 24 24"
+      >
         <path
-          d="M12 0c-6.626 0-12 5.373-12 12 0 5.302 3.438 9.8 8.207 11.387.599.111.793-.261.793-.577v-2.234c-3.338.726-4.033-1.416-4.033-1.416-.546-1.387-1.333-1.756-1.333-1.756-1.089-.745.083-.729.083-.729 1.205.084 1.839 1.237 1.839 1.237 1.07 1.834 2.807 1.304 3.492.997.107-.775.418-1.305.762-1.604-2.665-.305-5.467-1.334-5.467-5.931 0-1.311.469-2.381 1.236-3.221-.124-.303-.535-1.524.117-3.176 0 0 1.008-.322 3.301 1.23.957-.266 1.983-.399 3.003-.404 1.02.005 2.047.138 3.006.404 2.291-1.552 3.297-1.23 3.297-1.23.653 1.653.242 2.874.118 3.176.77.84 1.235 1.911 1.235 3.221 0 4.609-2.807 5.624-5.479 5.921.43.372.823 1.102.823 2.222v3.293c0 .319.192.694.801.576 4.765-1.589 8.199-6.086 8.199-11.386 0-6.627-5.373-12-12-12z">
+          d="M12 0c-6.626 0-12 5.373-12 12 0 5.302 3.438 9.8 8.207 11.387.599.111.793-.261.793-.577v-2.234c-3.338.726-4.033-1.416-4.033-1.416-.546-1.387-1.333-1.756-1.333-1.756-1.089-.745.083-.729.083-.729 1.205.084 1.839 1.237 1.839 1.237 1.07 1.834 2.807 1.304 3.492.997.107-.775.418-1.305.762-1.604-2.665-.305-5.467-1.334-5.467-5.931 0-1.311.469-2.381 1.236-3.221-.124-.303-.535-1.524.117-3.176 0 0 1.008-.322 3.301 1.23.957-.266 1.983-.399 3.003-.404 1.02.005 2.047.138 3.006.404 2.291-1.552 3.297-1.23 3.297-1.23.653 1.653.242 2.874.118 3.176.77.84 1.235 1.911 1.235 3.221 0 4.609-2.807 5.624-5.479 5.921.43.372.823 1.102.823 2.222v3.293c0 .319.192.694.801.576 4.765-1.589 8.199-6.086 8.199-11.386 0-6.627-5.373-12-12-12z"
+        >
         </path>
       </svg>
     </a>
-  </div>
-  <div class="border">
-    <a href={source} target="_blank" aria-details="Edit on GitHub" title="Edit on GitHub">✏</a>
-  </div>
-  <div class="border select">
-    <select name="theme">
-      <option value="light">🌖</option>
-      <option value="system">🌓</option>
-      <option value="dark">🌒</option>
-    </select>
+
+    <div class="select">
+      <select name="theme">
+        <option value="light">🌖</option>
+        <option value="system">🌓</option>
+        <option value="dark">🌒</option>
+      </select>
+    </div>
   </div>
 </header>
 
 <style scoped>
   header {
-    position: absolute;
-    top: 0;
-    left: 0;
     width: 100vw;
     height: var(--header-height);
     display: flex;
+    justify-content: space-between;
     align-items: center;
     gap: 3rem;
     padding-left: 2rem;
-    background-color: var(--bg1);
-    opacity: 0.7;
+    padding-right: 2rem;
+    z-index: 3;
+    transition: box-shadow 0.3s ease-in-out;
+  }
+
+  @media screen and (max-width: 1023px) {
+    header {
+      padding-left: 1rem;
+      padding-right: 1rem;
+    }
+  }
+
+  header.shadow {
+    box-shadow: 0 16px 32px -16px rgba(0, 0, 0, 0.1),
+      0 0 0 1px rgba(0, 0, 0, 0.1);
   }
 
   header * {
     background-color: transparent;
   }
 
+  .left-container,
+  .right-container {
+    display: flex;
+    align-items: center;
+    column-gap: 20px;
+  }
+
+  .burger {
```

Comment:
Fix mobile paddings pls (the top is higher right now)
![CleanShot 2023-05-23 at 05 22 50@2x](https://github.com/artalar/reatom/assets/27290320/362e21bf-43db-4d67-a703-cde6cf75d0fc)

Reply:
Done! Now looks better

![Screen Shot 2023-05-23 at 11 10 07](https://github.com/artalar/reatom/assets/25587090/473bd022-3064-4b4a-a7ac-f30818545177)